azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -309,6 +316,12 @@ public String getSlaOptionStr() {
     flowObj.put(FLOW_LOCK_ERROR_MESSAGE_PARAM, this.flowLockErrorMessage);
     flowObj.put(FLOW_DISPATCH_METHOD, getDispatchMethod().getNumVal());
 
+    if (this.getVersionSet() != null) {
```

Comment:
why use this.getVersionSet() instead of this.versionSet? I think latter is better.

Reply:
changed to this.versionSet()

Reply:
Done